src/pages/Home.js

```diff
@@ -6,10 +6,16 @@ import { db } from '../firebase/firebase'
 import { Link } from 'react-router-dom'
 import { motion } from 'framer-motion'
 import Layout from '../layouts/Layout'
+import TierSelect from '../components/form/TierSelect'
 
 const Home = () => {
   const [jobs, setJobs] = useState([])
 
+  const [tier, setTier] = useState(process.env.REACT_APP_ADVANCED_PLAN)
+  function receivingTierClick(e) {
```

Comment:
Just curious why the entire Event object is saved to state.

Also, if all `receivingTierClick` is doing is wrapping `setTier`, you can simply have the following below, `<TierSelect receivingTierClick={setTier} tier={tier} />`

Reply:
@nickytonline This is largely because this was the only way I knew to accomplish. I'd love to grab a virtual coffee and chat better ways if you ever a window!

Reply:
Sounds good. I’m free most days as I typically don’t have a lot of meetings so we can Zoom it up. Just hit me up in Slack.

Reply:
Ahh, I see why I got confused. `e` is not an [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event). It's the tier, e.g. "price_1GuKFPLy9mbkpBNAI6XtSdqT". If that's the case, you can still do as I suggest. We can talk more on Wednesday.